src/main/java/gov/usgs/earthquake/distribution/WebSocketListener.java

```diff
@@ -9,5 +9,5 @@
 public interface WebSocketListener {
   public void onMessage(String message);
```

Comment:
Add an onOpen method, in case listeners want to know.

Reply:
added and overridden to provide log output